packages/backend/src/services/category.ts

```diff
@@ -7,9 +8,16 @@ export class CategoryService {
     this._categoryRepository = categoryRepository;
   }
 
-  public getAll(query: Query) {
-    const take = query.limit ? +query.limit : 10;
+  public async getAll(req: Request) {
+    try {
+      const result = await this._categoryRepository.getAll(req.query);
 
-    return this._categoryRepository.getAll(take);
+      return result.map((item) => ({
+        ...item,
+        title: req.t(`categories.${item.title}`),
```

Comment:
please return title(or a key which can be used as identifier of FE side for filters, etc.)